docs/data-layer-api.md

```diff
@@ -0,0 +1,281 @@
+# Data Layer API Reference
+
+The [Data Layer API](/api-docs/store-management/data-layer-api) is a GraphQL management API that allows you to enable and disable the [Big Open Data Layer](/api-docs/analytics/bodl-for-storefronts) for a store.
+
+**Request URL:** 
+
+`https://api.bigcommerce.com/stores/{store_hash}/v3/...`
```

Comment:
@solofeed Do you have the request url?

Reply:
This is the graphql endpoint
`stores/{store_hash}/graphql`